webapp/src/App.svelte

```diff
@@ -2,83 +2,230 @@
   // import "../app.css";
   import Ferdous from "./components/Ferdous.svelte";
 
-  const key: string = import.meta.env.VITE_PUBLIC_STOCK_API_KEY;
-
-  interface stock {
-    name: string | void;
-    price: number | void;
-    pChange: number | void;
-  }
   let timeseries = "WEEKLY";
 
-  let graphData: { time: string; value: number }[] = [];
+  let graphData: { time: string; value: number; symbol: string }[] = [];
 
-  const trackedSymbols = ["QQQ"];
+  // Define the theme object here
+  const THEME = {
+    chart: {
+      layout: {
+        background: {
+            type: 'solid',
+            color: '#2B2B43',
+        },
+          lineColor: '#2B2B43',
+          textColor: 'white',
+          fontWeight: 'bold',
+      },
+      watermark: {
+          color: 'rgba(0, 0, 0, 0)',
+      },
+      crosshair: {
+          color: '#758696',
+      },
+      grid: {
+        vertLines: {
+            color: '#2B2B43',
+        },
+        horzLines: {
+            color: '#363C4E',
+        },
+      },
+    },
+    series: {
+        topColor: 'rgba(32, 226, 47, 0.56)',
+        bottomColor: 'rgba(32, 226, 47, 0.04)',
+        lineColor: 'rgba(32, 226, 47, 1)',
+    },
+  };
 
-  async function getStocks() {
-    console.log(graphData);
-  }
+  const trackedSymbols = ["QQQ", "PANW", "TSLA", "AAPL", "MSFT", "NVDA"];
```

Comment:
instead of hardcoding the symbols below for each graph. Do some sort of for loop of `trackedSymbols` to dynamically create the list of `Ferdous` components